src/themes/default/pages/Product.vue

```diff
@@ -15,7 +15,7 @@
           </div>
           <div class="col-md-5">
 
-            <h1 class="mb25 c-black"> {{ product.name | htmlDecode }} </h1>
+            <h1 class="mb25 c-black"> {{ configured_product.name | htmlDecode }} </h1>
```

Comment:
Not sure if this is best option; I mean - probably the expected behavior is that the Product Title is not changing on the product page when selecting variants; maybe let's have product.name here - intentionally

Reply:
`product.name` will also be dynamically changed, since we have a `configure` action in store. we should probably store the unchanged product somewhere.